parameters/us-west-2/dev.json

```diff
@@ -6,5 +6,6 @@
     "PublicAZBSubnetBlock=10.0.96.0/20",
     "PrivateAZCSubnetBlock=10.0.128.0/19",
     "PublicAZCSubnetBlock=10.0.160.0/20",
-    "HighlyAvailableNat=true"
+    "HighlyAvailableNat=true",
+    "EnableVpcFlowLogs=true"
```

Comment:
So, why is `EnableVpcFlowLogs=true` here but `false` in `us-east-1` above?

Reply:
Just to show an example that you could do either. Same reason why HA nat is True in us-west-2 but false in us-east-1. I was just following the existing pattern :)